pgmpy/tests/test_base/test_mixin_roles.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+import unittest
+
+from pgmpy.base import DAG
+
+
+class TestGraphRolesMixin(unittest.TestCase):
```

Comment:
The tests need to be in pytest format. We are trying to move our tests to the pytest format.